apilos_settings/api/api_views.py

```diff
@@ -164,27 +164,76 @@ def get(self, request):
             .values("statut")
             .annotate(total=Count("statut"))
         )
-        instruction = 0
-        a_signer = 0
-        signee = 0
-        for q in query_by_statuses:
-            if q["statut"] == ConventionStatut.INSTRUCTION:
-                instruction = q["total"]
-            if q["statut"] == ConventionStatut.A_SIGNER:
-                a_signer = q["total"]
-            if q["statut"] == ConventionStatut.SIGNEE:
-                signee = q["total"]
-
-        list_conv_kpi = [
-            ConvKPI(
-                "/conventions/?cstatut=2.+Instruction+requise",
-                instruction,
-                "En instruction",
-            ),
-            ConvKPI("/conventions/?cstatut=4.+A+signer", a_signer, "A signer"),
-            ConvKPI("/conventions/?cstatut=5.+Signée", signee, "Signées"),
-        ]
-
-        serializer = ConventionKPISerializer(list_conv_kpi, many=True)
 
-        return Response(serializer.data)
+        nb_conventions_by_status = {
+            convention_statut.value: 0 for convention_statut in ConventionStatut
+        }
+        nb_conventions_by_status.update(
+            {query["statut"]: query["total"] for query in query_by_statuses}
```

Comment:
💡 j'ai l'impression qu'on pourrait "fusionner" ces 2 lignes:
* soit via un _one-liner_  du style `query[convention_statut.value]['total] if convention_statut.value in query else 0` 
* soit un merge `nb_conventions_by-status = { convention_statut ... } | { query["statut"]} `

Qu'en pense-tu ?